src/main/java/frc/trigon/robot/subsystems/ampaligner/AmpAlignerConstants.java

```diff
@@ -0,0 +1,128 @@
+    private static final double GEAR_RATIO = 52;
+    static final boolean FOC_ENABLED = true;
+
+    private static final int MOTOR_AMOUNT = 1;
+    private static final DCMotor GEARBOX = DCMotor.getFalcon500Foc(MOTOR_AMOUNT);
+    private static final double AMP_ALIGNER_MASS_KILOGRAMS = 1.1;
+    public static final double AMP_ALIGNER_LENGTH_METERS = 0.52;
+    private static final Rotation2d
+            AMP_ALIGNER_MINIMUM_ANGLE = Rotation2d.fromDegrees(0),
+            AMP_ALIGNER_MAXIMUM_ANGLE = Rotation2d.fromDegrees(156);
+    private static final Rotation2d REVERSE_SOFT_LIMIT_THRESHOLD = Rotation2d.fromDegrees(0);
+    private static final boolean SIMULATE_GRAVITY = true;
+    private static final SingleJointedArmSimulation SIMULATION = new SingleJointedArmSimulation(
+            GEARBOX,
+            GEAR_RATIO,
+            AMP_ALIGNER_LENGTH_METERS,
+            AMP_ALIGNER_MASS_KILOGRAMS,
+            AMP_ALIGNER_MINIMUM_ANGLE,
+            AMP_ALIGNER_MAXIMUM_ANGLE,
+            SIMULATE_GRAVITY
+    );
+
+    static final SysIdRoutine.Config SYSID_CONFIG = new SysIdRoutine.Config(
+            Units.Volts.of(2).per(Units.Second.of(1)),
+            Units.Volts.of(4),
+            Units.Second.of(1000)
+    );
+
+    public static final Transform3d PITCHER_TO_AMP_ALIGNER = new Transform3d(0.5039, 0, 0.0434, new Rotation3d());
+
+    static final Rotation2d LIMIT_SWITCH_PRESSED_ANGLE = Rotation2d.fromDegrees(156);
+    static final double LIMIT_SWITCH_DEBOUNCE_TIME_SECONDS = 0.1;
+    static final Rotation2d ANGLE_TOLERANCE = Rotation2d.fromDegrees(0.3);
+
+    static {
+        configureMotor();
+    }
+
+    private static void configureMotor() {
+        final TalonFXConfiguration config = new TalonFXConfiguration();
+
+        config.Audio.BeepOnBoot = false;
+        config.Audio.BeepOnConfig = false;
+
+        config.MotorOutput.Inverted = INVERTED_VALUE;
+        config.MotorOutput.NeutralMode = NEUTRAL_MODE_VALUE;
+
+        config.HardwareLimitSwitch.ForwardLimitType = FORWARD_LIMIT_TYPE_VALUE;
+        config.HardwareLimitSwitch.ForwardLimitSource = FORWARD_LIMIT_SOURCE_VALUE;
+
+        config.Slot0.kP = P;
+        config.Slot0.kI = I;
+        config.Slot0.kD = D;
+        config.Slot0.kS = KS;
+        config.Slot0.kV = KV;
+        config.Slot0.kA = KA;
+        if (RobotHardwareStats.isSimulation())
+            config.Slot0.kG = KG;
+        config.Slot0.GravityType = GRAVITY_TYPE_VALUE;
+        config.Slot0.StaticFeedforwardSign = STATIC_FEEDFORWARD_SIGN_VALUE;
+
+        config.MotionMagic.MotionMagicAcceleration = MOTION_MAGIC_ACCELERATION;
+        config.MotionMagic.MotionMagicCruiseVelocity = MOTION_MAGIC_CRUISE_VELOCITY;
+
+        config.SoftwareLimitSwitch.ReverseSoftLimitEnable = true;
+        config.SoftwareLimitSwitch.ReverseSoftLimitThreshold = REVERSE_SOFT_LIMIT_THRESHOLD.getRotations();
+
+        config.Feedback.RotorToSensorRatio = GEAR_RATIO;
```

Comment:
this needs to be SensorToMechanismRatio, RotorToSensorRatio is only used for a fused cancoder